### PR TITLE
2.3.0-beta1 Release

### PR DESCRIPTION
**Pull request by kuychaco**
- [x] finalize release notes
- [x] release!

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,13 @@
 {
   "releases": {
+    "2.3.0-beta1": [
+      "[Fixed] Show commit warning messages only if changed files are present - #8795",
```
Comment (outofambit) [inline changelog.json]:
```suggestion
      "[Improved] Only show branch protection warning when changed files are present - #8795",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+      "[Fixed] Sign Windows installer using SHA256 digest - #8901",
```
Comment (outofambit) [inline changelog.json]: this isn't user-facing so i don't think we should include it
```suggestion
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+      "[Fixed] \"Do you want to fork this repository?\" Dialog - #8548",
```
Comment (outofambit) [inline changelog.json]:
```suggestion
      "[New] Prompt users to make a fork if they fail to push to a GitHub repository due to permissions  - #8548",
```

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,13 @@
...
+      "[Fixed] Add create fork flow entry point to commit warning - #8938",
```
Comment (outofambit) [inline changelog.json]:
```suggestion
      "[Added] Commit warning for repositories the user can't push to - #8938",
```

File: changelog.json [hunk 5]
```diff
@@ -1,5 +1,13 @@
...
+      "[Fixed] Open PR from fork in browser - #8928",
```
Comment (outofambit) [inline changelog.json]:
```suggestion
      "[Fixed] Showing pull request from a fork in browser uses the correct URL - #8928",
```

File: changelog.json [hunk 6]
```diff
@@ -1,5 +1,13 @@
...
+      "[Improved] Show an error when writing configuration file fails due to existing lock file"
```
Comment (outofambit) [inline changelog.json]:
```suggestion
      "[Improved] Show an error when editing a git configuration file fails due to existing lock file"
```

File: changelog.json [hunk 7]
```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.3.0-beta1": [
+      "[New] Prompt users to make a fork if they fail to push to a GitHub repository due to permissions  - #8548",
+      "[Added] Commit warning for repositories the user can't push to - #8938",
```
Comment (nerdneha) [inline changelog.json]:
```suggestion
      "[Added] Show a commit warning if the user can't push to the repository - #8938",
```

File: changelog.json [hunk 8]
```diff
@@ -1,5 +1,12 @@
...
+      "[Fixed] Showing pull request from a fork in browser uses the correct URL - #8928",
```
Comment (nerdneha) [inline changelog.json]:
```suggestion
      "[Fixed] Route the user to the correct URL when opening a pull request from a fork in the browser - #8928",
```

**Comment by outofambit** [comment]
will merge this since its now released!

**Pull request merged**